src/connectors/eggs-dom-inject.js

```diff
@@ -6,6 +6,9 @@
  */
 
 const isArtistPage = window.location.href.includes('/artist/');
+let frameID = '';
+let currentTime = 0;
+let duration = 180;
```

Comment:
Why did you set the initial duration value to 180?

Reply:
I set it pretty arbitrarily. It's around about the lower end of the typical length of a song. It's just there in case something were to completely break, it may fail a bit more gracefully. Unless something breaks, it will never actually be sent as the duration of a song.

Reply:
I think it should be `0` or `null`; the core will handle nullable duration.